modules/502-delivery/openapi/config-values.yaml

```diff
@@ -0,0 +1,151 @@
+type: object
+required:
+  - auth
+properties:
+  argocd:
+    type: object
+    default: {}
+    description: "ArgoCD configuration"
+    properties:
+      admin:
+        type: object
+        default: {}
+        properties:
+          enabled:
+            type: boolean
+            default: false
+            description: "Enable ArgoCD admin user"
+  storageClass:
```

Comment:
Does redis require a persistent volume? What kind of date argocd stores in it? What if we go without any persistent storage besides kubernetes?

Reply:
It caches repos. I don't know how it will work without cache.

https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#high-availability

```
Redis is only used as a throw-away cache and can be lost. When lost, it will be rebuilt without loss of service.
```